plugins/amazonq/chat/jetbrains-community/src/software/aws/toolkits/jetbrains/services/amazonq/QLoginWebview.kt

```diff
@@ -164,7 +164,7 @@ class QWebviewBrowser(val project: Project, private val parentDisposable: Dispos
             "reauth" -> {
                 ToolkitConnectionManager.getInstance(project).activeConnectionForFeature(QConnection.getInstance())?.let { conn ->
                     if (conn is ManagedBearerSsoConnection) {
-                        ApplicationManager.getApplication().executeOnPooledThread {
+                        pluginAwareExecuteOnPooledThread {
```

Comment:
maybe worth adding a lint rule to disallow `executeOnPooledThread` and its variants

Reply:
Ack - this is a good idea. I'll follow up with this in a separate PR.